### DISCUSSION TITLE
I'm raiding Prodigy!

### DISCUSSION THREAD

**Discussion by afkvido** [Prodigy-Hacking/ProdigyMathGameHacking#1346] [(A) Anything]
# I'm raiding Prodigy, Join me!

I've always wanted to do bad shit in Prodigy before hacks closed down, I'd like to promote a raid now.

Pretty much, follow the steps below, do that to cover all the NPCs. 

## Join the raid!

1. Go to Prodigy (with PHEx), 
2. Click the hack: `Morph Player`
3. Select the Morph Type: `Follow`
4. Select the Morph: `Horned Owl`
5. You'll now appear bugged, as an image file that stores sprites.
6. Anyways, that image has a massive hitbox, and about 5 of those can cover an entire zone, making it unclickable, they'll only be able to open your profile.
7. Keep in mind that this doesn't work on Harmony Island.

# Example

For example, I've covered most of Lamplight Town in Animalia right now.

<img width="1436" alt="Image" src="https://user-images.githubusercontent.com/69060894/161415600-f96a338b-f53a-4f63-ad45-c4459908b3c6.png">

**Comment by ghost** [comment]
> I'm raiding Prodigy!

Epic!
